doc/classes/ProjectSettings.xml

```diff
@@ -519,9 +519,21 @@
 		<member name="debug/shapes/collision/shape_color" type="Color" setter="" getter="" default="Color(0, 0.6, 0.7, 0.42)">
 			Color of the collision shapes, visible when "Visible Collision Shapes" is enabled in the Debug menu.
 		</member>
+		<member name="debug/shapes/navigation/agent_path_color" type="Color" setter="" getter="" default="Color(1, 0, 0, 1)">
+			Color to display enabled navigation agent paths, visible when "Visible Navigation" is enabled in the Debug menu and when an agent has debug enabled.
+		</member>
+		<member name="debug/shapes/navigation/agent_path_point_size" type="float" setter="" getter="" default="4.0">
+			Rasterized size (pixel) used to render navigation agent path points, visible when "Visible Navigation" is enabled in the Debug menu and when an agent has debug enabled.
+		</member>
 		<member name="debug/shapes/navigation/edge_connection_color" type="Color" setter="" getter="" default="Color(1, 0, 1, 1)">
 			Color to display edge connections between navigation regions, visible when "Visible Navigation" is enabled in the Debug menu.
 		</member>
+		<member name="debug/shapes/navigation/enable_agent_paths" type="bool" setter="" getter="" default="true">
+			If enabled, displays navigation agent paths when "Visible Navigation" is enabled in the Debug menu and when an agent has debug enabled.
+		</member>
+		<member name="debug/shapes/navigation/enable_agent_paths_xray" type="bool" setter="" getter="" default="true">
+			If enabled, displays navigation agent paths through geometry when "Visible Navigation" is enabled in the Debug menu and when an agent has debug enabled.
```

Comment:
debug paths are visible even if `Visible Navigation` is disabled - IMO that's better as `Visible Navigation` has just a single responsibility. Nevertheless, the descriptions here need to be aligned

Reply:
The Visible Navigation is toggling the NavigationServer3D debug_enabled bool. The bug was that this was not checked.